sklearn/ensemble/tests/test_stacking.py

```diff
@@ -646,6 +653,85 @@ def fit(self, X, y):
         stacker.n_features_in_
 
 
+@pytest.mark.parametrize("stack_method", ["auto", "predict", "predict_proba"])
```

Comment:
I am wondering if we should had the case of `"decision_function"` that could be used with `RidgeClassifier` and check the behaviour. Indeed, `decision_function` will output a numpy array of shape `(n_samples, n_targets)` (with the multilabel case) so it should not be a problem in regards to the implementation.

Reply:
Does `StackingClassifier` allows estimators to use `decision_function` as `stack_method`? Otherwise, in which scenario could this situation arise?

Reply:
The documentation says so and I think this possible looking at this part of the code: https://github.com/scikit-learn/scikit-learn/blob/36958fb24/sklearn/ensemble/_stacking.py#L128

Reply:
Ok got it, thanks :)